slsim/Sources/SourceVariability/light_curve_interpolation.py

```diff
@@ -0,0 +1,27 @@
+from scipy.interpolate import interp1d
+
+
+class LightCurveInterpolation(object):
+    """This class manages interpolation of light curve of a source."""
+
+    def __init__(self, times, magnitudes):
+        """
+        :param times: array of observation time
```

Comment:
this is the time without time dilatation (or the first arriving image), correct?

Reply:
I used MJD time and magnitude provided by @jpierel14 . I guess this is a the time without time dialation.